indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -183,10 +185,45 @@ class RHSubmitAbstractComment(RHAbstractBase):
     def _check_abstract_protection(self):
         return self.abstract.can_comment(session.user)
 
+    def _get_recipients(self):
+        judges = set()
+        conveners = set()
+        reviewers = set()
+
+        for entry in self.abstract.event.acl_entries:
+            if self.abstract.can_judge(entry.user):
+                judges.add(entry.user)
+            elif self.abstract.can_convene(entry.user):
+                conveners.add(entry.user)
+            elif self.abstract.can_review(entry.user):
+                reviewers.add(entry.user)
+        # Prevent sending duplicate emails for different roles
+        conveners -= judges
+        reviewers -= (judges | conveners)
+        return judges, conveners, reviewers
+
+    def _send_notification(self, recipients, comment):
+        for recipient in recipients:
+            tpl = get_template_module('events/abstracts/emails/comment.html', event=self.event, abstract=self.abstract,
+                                      submitter=session.user, comment=comment, recipient=recipient)
+            email = make_email(to_list=recipient.email, template=tpl, html=True)
+            send_email(email, self.event, 'Abstracts', session.user)
+
     def _process(self):
         form = AbstractCommentForm(abstract=self.abstract, user=session.user)
         if form.validate_on_submit():
             create_abstract_comment(self.abstract, form.data)
+            notify_on_new_comments = abstracts_reviewing_settings.get(self.event, 'notify_on_new_comments')
+            if notify_on_new_comments:
+                judges, conveners, reviewers = self._get_recipients()
```

Comment:
What do you think about passing the visibility of the comment to this method, and then returning a single set? I think the split return value is a leftover from when you used "Dear judge" etc, but that's no longer needed..